app/views/static_pages/invoice.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="container">
+  <div class="row justify-content-center">
+    <div class="col-12">
+      <div class="card card-default">
+        <div class="card-body">
+          <h3 id="card-text">Your charges</h3>
+          <hr>
+          <% @charges = Invoice.all.where(payment_id: current_person.user.payment.id)%>
+          <% @charges.each do |charge| %>
+            <div class="media mb-2">
+                <p><%= charge.description %></p>
+                <b><%= charge.amount %></b>
+                <p><%= charge.created_at %></p>
```

Comment:
Please format using ```charge.created_at.strftime("FORMAT")```. Here is a good resource: [https://www.foragoodstrftime.com/](https://www.foragoodstrftime.com/)